src/encoded/tests/test_upgrade_award.py

```diff
@@ -69,3 +69,12 @@ def test_award_upgrade_title_requirement(upgrader, award_5):
     value = upgrader.upgrade('award', award_5, target_version='6')
     assert value['title']
     assert value['schema_version'] == '6'
+
+
+def test_award_upgrade_milestones(upgrader, award_2):
+    award_2['schema_version'] = '6'
+    award_2['milestones'] = ['single-nuclei ATAC-seq', 'HiC']
```

Comment:
I think the code here is wrong. `milestones` for award is a list of objects rather than a list of strings.

Reply:
You are right, award upgrade code and test have been updated.